OxfordBodleian/Ullendorff/BDLeu8.xml

```diff
@@ -34,59 +38,230 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                                    in <ref target="#ms_i1"/>; elements of punctuation marks.
+                                    </seg>
                             </handNote>
                         </handDesc>
                         
                        <!-- <decoDesc>
                          
                         </decoDesc>-->
                         
-                   <!--     <bindingDesc>
+                        <additions>
+                            <list>
+                                <item xml:id="e1">
+                                    <desc>At least on one occasion, an erroneous character has been marked for deletion by dashes or lines above and below.</desc>
+                                </item>
+                                <item xml:id="e2">
+                                    <desc>On several of the miniatures, the words <foreign xml:lang="gez">ንግሥት፡ አዜብ፡</foreign> 
+                                        (<hi rend="italics">Nǝgǝśt ʾAzeb</hi>, "The Queen of (?) the South") have been written.
+                                    It is possible that this is the name of a secondary owner.</desc>
+                                </item>
+                            </list>
+                        </additions>
+                        
+                        <bindingDesc>
                             <binding xml:id="binding">
-                                <decoNote xml:id="b1"></decoNote>
-                                <decoNote xml:id="b2" type="bindingMaterial">
-                           <material key="wood"/>
-                        </decoNote>
+                                <decoNote xml:id="b1">The two parchment strips are sewn together with a narrow parchment strip.</decoNote>
+                                <decoNote xml:id="b2">A narrow parchment strip is attached through a hole in the middle of the upper part of the scroll.</decoNote>
+                                <decoNote xml:id="b3" type="SlipCase">
+                                    The manuscript is provided with a small slipcase of the kind typically used for codices.
+                                </decoNote>
                             </binding>
-                        </bindingDesc>-->
+                        </bindingDesc>
                         
                     </physDesc>
                     
-                <!--    <history>
+                <history>
                         <origin>
-                            <origDate when="" evidence=""/>
+                            <origDate notBefore="1800" notAfter="2002" evidence="lettering">nineteenth–twenty-first century</origDate>
+                            The manuscript can be dated palaeographically to the period between the
+                            nineteenth and the twenty-first century.
+                            The names <persName ref="PRS15041HaylaMaryam">Ḫayla Māryām</persName> and
```

Comment:
The information on names of the owners should be given in extras (and can then be very briefly repeated here without codicological details)